<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="translate_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_text_edit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>110</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_text_edit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="input_language_combo_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="output_language_combo_box">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Google Translate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
